.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: trusty
+language: java
+sudo: false
+
+jdk:
+  - oraclejdk8
+  - oraclejdk11
+  - openjdk8
+  - openjdk11
+
+cache:
+  directories:
+    - $HOME/.m2
+
+install:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+
+script:
+  - mvn verify -B -V
```

Comment:
is build (in install) and rebuild (in script) really needed? I believe doing it once is enough